system-tests/conftest.py

```diff
@@ -94,25 +131,40 @@ def run_containers(cmd, options):
     wait_until_kafka_ready(cmd, options)
 
 
-def build_and_run(options, request):
+def build_and_run(options, request, local_path=None, wait_for_debugger=False):
```

Comment:
Should we have a warning about using a debugger if `local_path=None`?

Reply:
done